src/nav/opentelemetry.yml

```diff
@@ -57,28 +57,22 @@ pages:
             path: /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-squid
           - title: StatsD
             path: /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-statsd
-  - title: Export OpenTelemetry to New Relic
+  - title: OpenTelemetry data in New Relic
     pages:
       - title: OTLP
         pages:
-          - title: Attribute length
-            path: /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-attributes
-          - title: Batching
-            path: /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-batching
-          - title: Compression
-            path: /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-compression
-          - title: OTLP versions
-            path: /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-versions
-      - title: Data mapping
-        pages:
-          - title: Logs
-            path: /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs
-          - title: Metrics
-            path: /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-metrics
-          - title: Resources
-            path: /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources
-          - title: Traces
-            path: /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-traces
+          - title: New Relic OTLP Endpoint
```

Comment:
```suggestion
          - title: New Relic OTLP endpoint
```

Reply:
@jack-berg I'd like to rename this page to better describe what the page is about. Some of my ideas are:
- Configure the OTLP endpoint 
- Send OTLP data to New Relic

Thoughts?

Reply:
Hmm.. The page encapsulates all the details about our OTLP endpoint. There are more details than simply configuration. For example, we explain which version of the protocol we support and some implementation details which are important for context.

I think "New Relic OTLP endpoint" is the best reflection of the pages content, but that "Send OTLP data to New Relic" keeps the door open for additional OTLP content outside of configuration. 
 Most of the content is about